src/config/statsd.ts

```diff
@@ -57,7 +57,8 @@ export const elapsedTimeMetrics = (
 	const method = req.method;
 
 	res.once("finish", () => {
-		const pathTag = req.route?.path || ((req.baseUrl || "/") + "*" );
+
+		const pathTag = req.baseUrl ? req.baseUrl + (req.route?.path || "*") : (req.route?.path || "/*");
```

Comment:
So the problem was that `req.route?.path` doesn't include base url. 

For example for request to `/api/resync` the `req.route.path` will be `/resync` and base url will be `/api`